org.exbin.BinEd.json

```diff
@@ -8,8 +8,9 @@
   "finish-args": [
      "--share=ipc",
      "--share=network",
-     "--socket=x11",
-     "--filesystem=home"
```

Comment:
this is NOT deprecated. Has never been.

Reply:
Well, I saw some message on flathub page so I assumed it's both legacy, but it was probably only meant for X11. 
![2024-01-06-213938_718x205_scrot](https://github.com/flathub/org.exbin.BinEd/assets/7817364/f43a3f3c-92ab-442c-a44a-198b9b2f0fab)
Is it worth switching back? It's not like app supports hex editing file drives or anything like that at this point...

Reply:
host is broader than home. so if your goal was to remove the warning, you went in the wrong direction.
as for legacy, yes it's about `x11`. but does it work on wayland? There is usually a reason.

Reply:
As a Java/Swing application it uses java 2D rendering API, where there are typically multiple renderers available (XWidow, Glib, OpenGL), but it depends on what particular version of JDK is and will be available in Flatpak. I believe there is Wayland renderer available for Java 20+, but I didn't try it myself, maybe it's just partial...

As the filesystem warning seems to be pretty much same, I'll probably keep it as is.

Reply:
the problem is that if it doesn't support wayland this will break for users that use Wayland.

Reply:
in short it shouldn't have been changed without proper consideration.

Reply:
That's why I also included fallback-x11. Well, it's true I find it somewhat unlikely they will finish Wayland support for Java anytime soon to the level that it will get backported to JDK 11 and flatpak with update, but I was like it could happen in next few years before next update on BinEd - so it was just to be sure :-|

Therefore I believe it should work as is, but if you insist, I'm willing to remove wayland for now...

Reply:
> That's why I also included fallback-x11

You don't understand what fallback-x11 does. 

What it does is that if the user doesn't have wayland, then x11 is exposed. But otherwise it's not. So on wayland there is no X11 by default and you Java program with fail.